package.json

```diff
@@ -6,43 +6,43 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
-    "lint": "ng lint",
+    "lint": "gts lint",
     "e2e": "ng e2e",
     "check": "gts check",
     "clean": "gts clean",
-    "compile": "tsc -p .",
+    "compile": "tsc",
     "fix": "gts fix",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
-    "posttest": "npm run check"
+    "posttest": "npm run lint"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.2.0",
+    "@angular/animations": "~10.1.5",
     "@angular/cdk": "^8.1.2",
-    "@angular/common": "~8.2.0",
-    "@angular/compiler": "~8.2.0",
-    "@angular/core": "~8.2.0",
-    "@angular/forms": "~8.2.0",
-    "@angular/platform-browser": "~8.2.0",
-    "@angular/platform-browser-dynamic": "~8.2.0",
-    "@angular/router": "~8.2.0",
+    "@angular/common": "~10.1.5",
+    "@angular/compiler": "~10.1.5",
+    "@angular/core": "~10.1.5",
+    "@angular/forms": "~10.1.5",
+    "@angular/platform-browser": "~10.1.5",
+    "@angular/platform-browser-dynamic": "~10.1.5",
+    "@angular/router": "~10.1.5",
     "core-js": "^2.5.4",
     "moment": "^2.22.2",
-    "rxjs": "~6.3.3",
-    "tslib": "^1.9.0",
-    "zone.js": "~0.8.26"
+    "rxjs": "~6.6.3",
+    "tslib": "^2.0.0",
+    "zone.js": "~0.10.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.802.0",
-    "@angular/cli": "^8.2.0",
-    "@angular/compiler-cli": "~8.2.0",
-    "@angular/language-service": "~8.2.0",
-    "@types/node": "~8.9.4",
+    "@angular-devkit/build-angular": "^0.1001.6",
+    "@angular/cli": "^10.1.6",
+    "@angular/compiler-cli": "~10.1.5",
+    "@angular/language-service": "~10.1.5",
+    "@types/node": "^14.11.2",
     "clang-format": "^1.2.4",
-    "gts": "^0.9.0",
+    "gts": "^3.0.0",
```

Comment:
nit: can you install the `prettier-plugin-organize-imports` package as well? it will make sure that prettier alphabetizes import statements (something clang-format did)